OneBusAway/ui/trip_details/VehicleMapController.swift

```diff
@@ -260,4 +254,58 @@ class VehicleMapController: UIViewController, MKMapViewDelegate {
 
         return annotationView
     }
+    
+    // MARK: - UI Configurations
+    
+    func createToggleHoverBar() {
+        let blurContainer = OBAVibrantBlurContainerView.init(frame: CGRect.zero)
+        self.view.addSubview(blurContainer)
+        blurContainer.snp.makeConstraints { (make) in
+            make.width.equalTo(40)
+            make.height.equalTo(30)
+            make.right.bottom.equalToSuperview().offset(-OBATheme.defaultPadding)
+        }
+        
+        blurContainer.vibrancyEffectView.addSubview(self.toggleButton)
+        self.toggleButton.snp.makeConstraints { make in
+            make.edges.equalToSuperview()
+        }
+        self.toggleButton.addTarget(self, action: #selector(toggleButtonTapped), for: .touchUpInside)
+        self.toggleButton.isSelected = self.expanded
+        
+    }
+    
+    
+
+    func createLocationHoverBar() {
+        let recenterMapButton = UIBarButtonItem(image: UIImage(named: "Map_Selected"), landscapeImagePhone: nil, style: .plain, target: self, action: #selector(self.recenterMap))
+        
+        //instantiate and customize location hover bar
+        let locationHoverBar = ISHHoverBar()
```

Comment:
I apologize for not being clear enough about this: delete the blur container and stick an `ISHHoverBar` with two buttons in its place.